src/main/java/org/folio/services/budget/BudgetService.java

```diff
@@ -102,35 +107,42 @@ public Future<Void> deleteBudget(String id, RequestContext requestContext) {
   }
 
   private void validateBudget(SharedBudget budget) {
+    log.debug("validateBudget:: Validation budget: {}", budget.getId());
     List<Error> errors = new ArrayList<>();
 
     errors.addAll(checkRemainingEncumbrance(budget));
     errors.addAll(checkRemainingExpenditure(budget));
 
     if (!errors.isEmpty()) {
+      log.error("validateBudget:: Error found during validation for budget: {}", budget.getId());
```

Comment:
it would be more useful if we add errors to the log, you can use this example
https://github.com/folio-org/mod-orders/pull/833/files#diff-4239a3815b4994127661b5435e8a7eaf09dd9f5fcd501817f8629fac77e4233cR112

Reply:
added, thanks

Reply:
added logs, thanks